pkgs/development/python-modules/uproot/default.nix

```diff
@@ -0,0 +1,22 @@
+{lib, fetchPypi, buildPythonPackage, numpy }:
+
+buildPythonPackage rec {
+  pname = "uproot";
+  version = "2.8.18";
+  name = "${pname}-${version}";
```

Comment:
drop this line

Reply:
you mean the `name` one? If I do I get an error. Am I using a too old version of `buildPythonPackage`?

Reply:
`buildPython*` has been setting `name` automatically for quite some time. Which error are you getting?